README.md

```diff
@@ -1,28 +1,85 @@
 # Introduction to Machine Learning for the Arts
 
+## OUTLINE
 
-## COURSE DESCRIPTION 
+### 1: Intro!
+* Session 1 (M 9/3): Introduction to Machine Learning
 
-An introductory course designed to provide students with hands-on experience developing creative coding projects with machine learning. The history, theory, and application of machine learning algorithms and related datasets are explored in a laboratory context of experimentation and discussion. Examples and exercises will be demonstrated in JavaScript using the p5.js, ml5.js, and TensorFlow.js libraries. In addition, students will learn to work with open source pre-trained models in the cloud using Runway. Principles of data collection and ethics are introduced. Weekly assignments, team and independent projects, and project reports are required. 
+### 2: Machine Learning Models 1
+* Session 1 (M 9/9): Image Classification
+* Session 2 (W 9/11): Transfer learning
+
+### 3: Machine Learning Models 2
+* Session 1 (M 9/16): Pre-trained models
+* Session 2 (W 9/18): Physical Interaction
+
+### 4: Training a Model
+* Session 1 (M 9/23): Data Collection
+* Session 2 (W 9/25): Model Training
+
+### 5: Real-Time Data / Interaction
+* Session 1 (M 10/8): Real-Time Data Collection
+* Session 2 (W 10/2): Real-Time Model Training
+
+### 6: Convolutional Neural Networks
+* Session 1 (M 10/7): Image Data
+* Session 2 (W 10/9): Model Training
+
+### 7: Recurrent Neural Networks
+* Session 1 (T 10/15): SketchRNN (Drawing)
+* Session 2 (W 10/16): Magenta (Sound)
+
+### 8: Training a RNN (Python)
+* Session 1 (M 10/21): Data Collection and Python Environment
+* Session 2 (W 10/23): Training and Deployment
+
+### 9: Pre-Trained Models with RunwayML
+* Session 1 (M 10/28): Runway Basics
+* Session 2 (W 10/30): Inputs/Outputs and Networking
+
+### 10: Generative Models with Runway
+* Session 1 (M 11/4): Generative Adversarial Networks
+* Session 2 (W 11/6): Interactive Image Synthesis
+
+### 11: Final Project Proposals
+* Session 1 (M 11/11): Group 1
+* Session 2 (M 11/13): Group 2
+
+### 12: Final Project Development
+* TBD (M 11/18)
+* TBD (W 11/20)
+* TBD (M 11/25)
+
+### 13: User Testing
```

Comment:
The user testing week was excellent in the Nature of Code course. Definitely +💯 to this!